.github/workflows/bump-cnspec.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger cnquery bump in cnspec
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Ben just discovered that this should only happen on release. This way we would avoid that we have accidental pushes to other repos

Reply:
Thanks. I changed this.

I'm not sure what you mean by "pushes to other repos". This workflow does not (git) push anything. It dispatches an event to the other repo and will there trigger a workflow.

Reply:
Thanks, looks great.